llvm/lib/Transforms/Utils/LCSSA.cpp

```diff
@@ -374,6 +391,8 @@ bool llvm::formLCSSA(Loop &L, const DominatorTree &DT, const LoopInfo *LI,
 
   SmallVector<BasicBlock *, 8> ExitBlocks;
   L.getExitBlocks(ExitBlocks);
```

Comment:
This getExitBlocks() call should also be served from the cache.

Reply:
woops, I attempted to do that (the next 2 lines) but my modification was incomplete. Should be fixed now

Reply:
Did a little bit more cleanup here to make this more closely match the other getExitBlocks() callsite and save and pass down the ExitBlocks as a SmallVectorImpl